dist/migration/migrate-page/migrate-page.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Migrate Page
+ *
+ * @package Atomic Blocks
+ */
+
+/**
+ * Load Migrate Page styles in the admin
+ *
+ * @since 1.0.0
+ * @param string $hook The current admin page.
+ */
+function atomic_blocks_migrate_load_admin_scripts( $hook ) {
+	if ( 'atomic-blocks_page_atomic-blocks-migrate-page' !== $hook ) {
+		return;
+	}
+
+	/**
+	 * Load scripts and styles
+	 *
+	 * @since 1.0
+	 */
+
+	// Migrate Page styles.
+	wp_register_style( 'atomic-blocks-migrate', plugins_url( '/migrate-page/ab-migrate.css', dirname( __FILE__ ) ), false, '1.0.0' );
+	wp_enqueue_style( 'atomic-blocks-migrate' );
+}
+add_action( 'admin_enqueue_scripts', 'atomic_blocks_migrate_load_admin_scripts' );
+
+/**
+ * Adds a submenu item for the Migrate Page under the Atomic Blocks top level menu.
+ *
+ * @since 1.0.0
+ */
+function atomic_blocks_migrate_menu() {
+
+	add_submenu_page(
```

Comment:
Thanks @kienstra. 

What are your thought on this filter @mindctrl ?

Reply:
Thinking on this further, this filter would really only be helpful it it was used prior to updating Atomic Blocks by the developer. Otherwise the notice appears immediately after updating the plugin. 

Since the dev's action has to be taken prior to the plugin being updated, they could also do this to simply prevent updates to Atomic Blocks until they are ready to migrate:

```
add_filter('site_transient_update_plugins', 'remove_update_notification');
function remove_update_notification($value) {
     unset($value->response["atomic-blocks/atomicblocks.php"]);
     return $value;
}
```

Thoughts @kienstra ?

Reply:
Hi @johnstonphilip,
Ah, good point. The filter you added should be fine to opt-out of the migration.